content/sdk/10.js/00.ethers/20.v6/03.providers.md

```diff
@@ -1,104 +1,69 @@
 ---
 
 A Web3 Provider object provides application-layer access to underlying blockchain networks.
 
-The [`zksync-ethers`](https://www.npmjs.com/package/zksync-ethers) library supports provider methods from
-the [`ethers.js`](https://docs.ethers.io/v6/api/providers) library and supplies additional functionality.
+The [`zksync-ethers`](https://www.npmjs.com/package/zksync-ethers/v/5.0.0) library supports
+provider methods from the [`ethers.js`](https://docs.ethers.io/v6/api/providers) library and
+supplies additional functionality.
 
-## Available providers
+Two providers are available:
 
-- **Provider**: Extends [`ethers.JsonRpcProvider`](https://docs.ethers.org/v6/api/providers/jsonrpc/#about-jsonrpcProvider)
-with ZKsync specific methods.
-- **BrowserProvider**: Extends the ZKsync Era `Provider` class for better compatibility with Web3 wallets.
+- [`Provider`](/sdk/js/ethers/v6/providers#provider): Supplies the same functionality as
+  [`ethers.providers.JsonRpcProvider`](https://docs.ethers.org/v6/api/providers/jsonrpc-provider/#JsonRpcProvider)
+  and extends it with ZKsync-specific methods.
+- [`Web3Provider`](/sdk/js/ethers/v6/providers#web3provider): Extends the ZKsync
+Era [`Provider`](/sdk/js/ethers/v6/providers#provider) class to
+make it more compatible
+  with Web3 wallets.
 
 ::callout{icon="i-heroicons-light-bulb"}
-Use `BrowserProvider` for browser integrations.
+Use the [`Web3Provider`](/sdk/js/ethers/v6/providers#web3provider) for browser integrations.
 ::
 
 ## `Provider`
 
 ### `constructor`
 
-Creates a zksync Era `Provider` object.
+Returns a ZKsync Era `Provider` object.
 
-**Parameters:**
+#### Inputs
 
-| Parameter  | Type                                                                                 | Description                                                |
-| ---------- | ------------------------------------------------------------------------------------ | ---------------------------------------------------------- |
-| `url?`     | [`ethers.FetchRequest`](https://docs.ethers.org/v6/api/utils/fetching/#FetchRequest) | Network RPC URL (optional).                                |
-| `network?` | [`ethers.Networkish`](https://docs.ethers.org/v6/api/providers/#Networkish)          | Network name, chain ID, or object with network details.     |
-| `options?` | `any`                                                                                | Additional provider options (optional).                    |
+| Parameter  | Type                                                                                   | Description                |
+| ---------- | -------------------------------------------------------------------------------------- | -------------------------- |
+| `url?`     | string or [`ConnectionInfo`](https://docs.ethers.org/v6/api/utils/web/#ConnectionInfo) | Network RPC URL |
+| `network?` | `ethers.providers.Networkish`                                                          | Network name    |
```

Comment:
Is there a reason why you removed the link to the ethers doc for network as well as removed part of the description for it?